### PR TITLE
FIX svm.py, svr.py + add expe SVM for hypergradient computation

### PR DESCRIPTION
**Pull request by QB3**

### CODE REVIEW

File: expes/hypergradient/plot_hypergradient_svm.py [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+    for method in methods:
+        print("Dataset %s, maxit %i" % (method, max_iter))
+        for i in range(2):
+            monitor = Monitor()
+            model = SVM(max_iter=max_iter)
+            criterion = HeldOutSmoothedHinge(idx_train, idx_val)
+            if method == "sota":
+                clf = LinearSVC(
+                    C=np.exp(logC), loss='hinge', max_iter=max_iter, tol=1e-32,
+                    permute=False)
```
Comment (agramfort) [inline expes/hypergradient/plot_hypergradient_svm.py:66]: No need to put fit_intercept=False ?

Comment (QB3) [reply expes/hypergradient/plot_hypergradient_svm.py:75]: I might have missed something, but a priori there is no fit intercept=True to pass to the `LinearSVC` from Lightning:
https://contrib.scikit-learn.org/lightning/generated/lightning.classification.LinearSVC.html

**Comment by agramfort** [comment]
ok I had not realized it the lightning class and not sklearn

>

**Comment by mathurinm** [comment]
I have merged master and fixed conflicts.

As this fixes bug in the main code I'm in favor of merging it as soon as it's green, and finishing the expe in another PR.

**Review by agramfort — APPROVED** [review]

**Pull request merged**